manifest-schema.graphql

```diff
@@ -27,14 +27,25 @@ type EthereumContractMapping {
   file: File!
   entities: [String!]!
   abis: [EthereumContractAbi!]!
-  eventHandlers: [EthereumContractEventHandlers]!
+  blockHandlers: EthereumBlockHandler
```

Comment:
Also, maybe this should be singular as well since it is not an array?

Reply:
Yep, good point